dashboard/test/models/user_test.rb

```diff
@@ -744,7 +744,7 @@ def complete_script_for_user(user, script, completed_date = Time.now)
   end
 
   test "needs_to_backfill_user_scripts?" do
-    user = create :student
```

Comment:
maybe also add a test to cover that it returns false where the user is newer but otherwise would `needs_to_backfill_user_scripts`